### ISSUE TITLE
exiftool fails to install with 404

### ISSUE THREAD

**Issue by yaakov-h** [Homebrew/homebrew-core#75399]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
yaakov@Ilum ~ % brew config
HOMEBREW_VERSION: 3.1.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: bc9b98aa0ba777889218ac51f67ccc76b0be6a28
Last commit: 4 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 5e0db24ed9eac01feb8a672bf0056f3f7f631d2a
Core tap last commit: 37 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 12.0 build 1200
Git: 2.24.3 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.2.3-arm64
CLT: 12.4.0.0.1.1610135815
Xcode: 12.4
Rosetta 2: false
```

```shell
yaakov@Ilum ~ % brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libndi.4.dylib

Warning: Homebrew's "sbin" was not found in your PATH but you have installed
formulae that put executables in /opt/homebrew/sbin.
Consider setting your PATH for example like so:
  echo 'export PATH="/opt/homebrew/sbin:$PATH"' >> ~/.zshrc
```

### 

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

Install `exiftool`

### What happened (include all command output)?

```shell
yaakov@Ilum ~ % brew install exiftool
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 3 taps (homebrew/core, homebrew/cask and homebrew/services).
==> New Formulae
cadence-workflow        dua-cli                 kickstart               net-tools               six                     zinit
conftest                enkits                  minisat                 organize-tool           systemd                 zsh-vi-mode
crackpkcs               epr                     mx                      pcp                     xplr
==> Updated Formulae
Updated 1191 formulae.
==> Renamed Formulae
fcct -> butane
==> New Casks
battery-buddy           command-pad             dingtalk-lite           logseq                  nordlocker              pop
calmly-writer           devilutionx             katrain                 magicplot               nordpass                wxmacmolplt
==> Updated Casks
Updated 804 casks.
==> Deleted Casks
adobe-air-sdk                flash-npapi                  flash-player-debugger-npapi  jira-client                  magicplotstudent
battery-guardian             flash-player                 flash-player-debugger-ppapi  kk7ds-python-runtime         mp3tag
beatport-pro                 flash-player-debugger        flash-ppapi                  magicplotpro                 screen

==> Downloading https://homebrew.bintray.com/exiftool-12.16.arm64_big_sur.bottle.tar.gz
#=#=-#  #                                                                     
curl: (22) The requested URL returned error: 404 Not Found
Trying a mirror...
==> Downloading https://ghcr.io/v2/homebrew/core/exiftool-12.16.arm64_big_sur.bottle.tar.gz
==> Downloading from https://github.com/-/v2/packages/container/package/homebrew%2Fcore%2Fexiftool-12.16.arm64_big_sur.bottle.tar.gz

curl: (22) The requested URL returned error: 404 
Error: Failed to download resource "exiftool"
Download failed: https://ghcr.io/v2/homebrew/core/exiftool-12.16.arm64_big_sur.bottle.tar.gz
```

### What did you expect to happen?

`exiftool` to be installed

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
Run `brew install exiftool`
```

**Comment by carlocab** [comment]
Please do `brew update` and try again.

**Comment by carlocab** [comment]
@Bo98 is this supposed to have been fixed by https://github.com/Homebrew/brew/pull/11162?

**Comment by yaakov-h** [comment]
```
yaakov@Ilum Downloads % brew update
Already up-to-date.
yaakov@Ilum Downloads % brew install exiftool
==> Downloading https://ghcr.io/v2/homebrew/core/exiftool/manifests/12.16
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/exiftool/blobs/sha256:a0d8370ee8da32d13b5deb9acfbfa5e460a17c248b358f44c026690a8d22905f
==> Downloading from https://pkg-containers-az.githubusercontent.com/ghcr1/blobs/sha256:a0d8370ee8da32d13b5deb9acfbfa5e460a17c248b358f44c026690a8
######################################################################## 100.0%
==> Reinstalling exiftool 
==> Pouring exiftool--12.16.arm64_big_sur.bottle.tar.gz
🍺  /opt/homebrew/Cellar/exiftool/12.16: 569 files, 22.9MB
```

what is this sorcery?!

**Comment by yaakov-h** [comment]
(full disclosure: I use `brew install --build-from-source exiftool` in the meantime but that doesn't seem to have output anything that indicates that it ran an update as part of that process)

<details>

```
yaakov@Ilum ~ % brew install --build-from-source exiftool
==> Downloading https://exiftool.org/Image-ExifTool-12.16.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL
==> make all
🍺  /opt/homebrew/Cellar/exiftool/12.16: 569 files, 22.9MB, built in 5 seconds
==> `brew cleanup` has not been run in 30 days, running now...
Removing: /opt/homebrew/Cellar/aom/2.0.2... (22 files, 6.5MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/aom--2.0.2.arm64_big_sur.bottle.tar.gz... (2.7MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/autoconf--2.69.arm64_big_sur.bottle.4.tar.gz... (872.3KB)
Removing: /opt/homebrew/Cellar/cairo/1.16.0_4... (119 files, 6MB)
Removing: /opt/homebrew/Cellar/dav1d/0.8.1... (15 files, 988.9KB)
Removing: /Users/yaakov/Library/Caches/Homebrew/ffmpeg--4.3.2_4.arm64_big_sur.bottle.tar.gz... (18.6MB)
Removing: /opt/homebrew/Cellar/gdbm/1.18.1_1... (25 files, 876.2KB)
Removing: /Users/yaakov/Library/Caches/Homebrew/glib--2.68.0.arm64_big_sur.bottle.tar.gz... (6.5MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/glib--2.66.7.arm64_big_sur.bottle.tar.gz... (6.4MB)
Removing: /opt/homebrew/Cellar/gobject-introspection/1.66.1_1... (191 files, 12.8MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/gobject-introspection--1.66.1_1.arm64_big_sur.bottle.tar.gz... (1.8MB)
Removing: /opt/homebrew/Cellar/icu4c/67.1... (258 files, 72.4MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/kubernetes-cli--1.20.4_1.arm64_big_sur.bottle.tar.gz... (11.4MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/libassuan--2.5.4.arm64_big_sur.bottle.tar.gz... (159.9KB)
Removing: /Users/yaakov/Library/Caches/Homebrew/libbluray--1.2.1.arm64_big_sur.bottle.1.tar.gz... (347.3KB)
Removing: /opt/homebrew/Cellar/libffi/3.3_2... (17 files, 617.4KB)
Removing: /Users/yaakov/Library/Caches/Homebrew/libgpg-error--1.41.arm64_big_sur.bottle.tar.gz... (309.6KB)
Removing: /Users/yaakov/Library/Caches/Homebrew/libksba--1.5.0.arm64_big_sur.bottle.tar.gz... (169.4KB)
Removing: /opt/homebrew/Cellar/libtool/2.4.6_2... (71 files, 3.7MB)
Removing: /opt/homebrew/Cellar/libvpx/1.9.0... (17 files, 1.7MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/libvpx--1.9.0.arm64_big_sur.bottle.tar.gz... (744.5KB)
Removing: /Users/yaakov/Library/Caches/Homebrew/libwebsockets--4.1.6.arm64_big_sur.bottle.tar.gz... (628.6KB)
Removing: /Users/yaakov/Library/Caches/Homebrew/mosquitto--2.0.9.arm64_big_sur.bottle.tar.gz... (451.3KB)
Removing: /opt/homebrew/Cellar/nettle/3.7... (89 files, 2.8MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/node--15.13.0.arm64_big_sur.bottle.tar.gz... (14.7MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/oniguruma--6.9.6.arm64_big_sur.bottle.tar.gz... (433.6KB)
Removing: /opt/homebrew/Cellar/openssl@1.1/1.1.1j... (8,071 files, 18MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/openssl@1.1--1.1.1j.arm64_big_sur.bottle.tar.gz... (5.2MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/protobuf--3.15.6.arm64_big_sur.bottle.tar.gz... (4.5MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/python@3.9--setuptools--53.0.0.tar.gz... (2.0MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/python@3.9--3.9.2.tar.xz... (18MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/python@3.9--3.9.2_4.arm64_big_sur.bottle.tar.gz... (13.6MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/rav1e--0.4.0.arm64_big_sur.bottle.tar.gz... (16.5MB)
Removing: /opt/homebrew/Cellar/rubberband/1.9.0... (12 files, 616.4KB)
Removing: /Users/yaakov/Library/Caches/Homebrew/sqlite--3.35.3.arm64_big_sur.bottle.tar.gz... (1.9MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/terraform--0.14.9.arm64_big_sur.bottle.tar.gz... (17.2MB)
Removing: /Users/yaakov/Library/Caches/Homebrew/x264--r3048.arm64_big_sur.bottle.tar.gz... (1.9MB)
Removing: /opt/homebrew/Cellar/x265/3.4_2... (11 files, 6.6MB)
Removing: /Users/yaakov/Library/Logs/Homebrew/mosh... (64B)
Removing: /Users/yaakov/Library/Logs/Homebrew/azure-cli... (64B)
Removing: /Users/yaakov/Library/Logs/Homebrew/smartmontools... (64B)
Removing: /Users/yaakov/Library/Logs/Homebrew/libyaml... (64B)
Removing: /Users/yaakov/Library/Logs/Homebrew/openssl@1.1... (64B)
Removing: /Users/yaakov/Library/Logs/Homebrew/socat... (64B)
Removing: /Users/yaakov/Library/Logs/Homebrew/python@3.8... (12 files, 4.6MB)
Removing: /Users/yaakov/Library/Logs/Homebrew/ser2net... (64B)
Removing: /Users/yaakov/Library/Logs/Homebrew/protobuf... (64B)
Pruned 0 symbolic links and 4 directories from /opt/homebrew
```

</details>

**Comment by Bo98** [comment]
> @Bo98 is this supposed to have been fixed by [Homebrew/brew#11162](https://github.com/Homebrew/brew/pull/11162)?

Yes. It was released with 3.1.3 today.

**Comment by carlocab** [comment]
Whoops; thought it was already in a release tag. Thanks again Bo.
